e2e/e2etests/e2etests.go

```diff
@@ -98,6 +100,11 @@ var AllE2ETests = []runner.E2ETest{
 		"withdraw BTC from ZEVM",
 		TestBitcoinWithdraw,
 	},
+	{
+		TestBitcoinWithdrawRestrictedName,
```

Comment:
This is some kind of niche test case. I would leave it at the bottom to keep the more general test first.

Not a blocker though, I think we can create an issue specifically to reorder all tests

Reply:
https://github.com/zeta-chain/node/pull/1791/commits/ce021e113520dbab64e1d27e5512c968b2dcefff